scripts/update.py

```diff
@@ -0,0 +1,129 @@
+import json, shutil, subprocess
+
+from github import Github
+
+from releases import get_latest_releases, SAMPLES_REPO
+from util import get_path_from_version_id, get_version_parts, write_to_github_output
+from docs import get_docs_update
+
+from constants import Version, LINE, ROOT, TMP_PATH, TAGS_PATH, TITLES, IS_ACTIONS
+
+g = Github()
+
+def do_versioned_commits(updates: list[tuple[str, str]], msg: str) -> None:
+  """
+  Performs two commits to show the git diff.
+  :param updates: A list of tuples of the form (old_version, new_version)
+  :param msg: The commit message
+  """
+
+  if not IS_ACTIONS:
+    print('Not in GitHub Actions, skipping commit.')
+    print(f'Updates: {updates}')
+    print(f'Message: {msg}')
+    return
+
+  for prev, new in updates:
+    print(f'Copying {prev} to {new} for versioning commit...')
+
+    prev_path = ROOT / get_path_from_version_id(prev)
+    new_path = ROOT / get_path_from_version_id(new)
+
+    if not prev_path.exists():
+      print(f'Previous version path {prev_path} does not exist')
+      exit(1)
+
+    shutil.copytree(prev_path, new_path, dirs_exist_ok=True)
+
+    print(f'Copied documentation files from {prev_path} to {new_path}')
+
+  copy_previous_version_msg = f'Copy previous version files for "{msg}"'
+  print(f'Committing "{copy_previous_version_msg}"')
+
+  # add previous files commit
+  subprocess.run(['git', 'add', '--all'], cwd=ROOT)
+  subprocess.run(['git', 'commit', '-m', copy_previous_version_msg], cwd=ROOT)
+
+  # copy new files
+  shutil.copytree(TMP_PATH, ROOT, dirs_exist_ok=True)
+
+  final_msg = f'Docs update: {msg}'
```

Comment:
Don't like this `final_message` variable. Just inline that into your print statement.

Reply:
It is used twice (debug log, commit message) like the previous `copy_previous_version_msg`